lib/rubocop/cop/rspec/excessive_docstring_spacing.rb

```diff
@@ -66,7 +66,9 @@ def excessive_whitespace?(text)
 
         # @param text [String]
         def strip_excessive_whitespace(text)
-          text.strip.gsub(/[[:blank:]]{2,}/, ' ')
+          text
+            .gsub(/[[:blank:]]{2,}/, ' ')
+            .sub(/\A[[:blank:]]|[[:blank:]]\z/, '')
```

Comment:
question: what happens if it has both leading and trailing whitespace? will this remove them both or do we need to `.sub` calls to achieve that?

Reply:
They're both removed, but it takes more than one iteration. There's not really a good reason to keep this `sub` instead of `gsub`, so I'll switch this to a `gsub`, so it only takes one iteration.

When writing this, I was also curious about this, and wrote a quick test and saw it pass. I removed it to be consistent with the current file (since there weren't any other combined leading/trailing whitespace tests). Since I was curious about this when I wrote it, and you were curious about it when you read it, it probably makes sense to keep those in. Locally, each of those tests takes about 0.004 seconds, so total increase of about a hundredth of a second to the test suite seems okay—let me know if you disagree.